app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Please modify the views in devise for the `registration` and `login` to match the wireframes and their styling.
You can take a look at the wireframes in the [sneak peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md) of the project.

Your Implementation| Wireframe
---|---
<img width="1230" alt="Screenshot 2023-10-04 at 5 55 54 PM" src="https://github.com/souad988/the-x-blog/assets/44798044/3d30b265-761c-4327-bea6-e36e58c7279b">|![blog_login](https://github.com/souad988/the-x-blog/assets/44798044/7aa9ddde-6e63-4701-9f76-8b035fe5ca40)